tensorboard/uploader/uploader_test.py

```diff
@@ -75,6 +75,25 @@ def test_create_experiment(self):
         eid = uploader.create_experiment()
         self.assertEqual(eid, "123")
 
+    def test_create_experiment_with_name_and_description(self):
+        logdir = "/logs/foo"
+        mock_client = _create_mock_client()
+        uploader_name = uploader_lib.TensorBoardUploader(
+            mock_client, logdir, name="name"
+        )
+        uploader_description = uploader_lib.TensorBoardUploader(
+            mock_client, logdir, description="description"
+        )
+        uploader_both = uploader_lib.TensorBoardUploader(
+            mock_client, logdir, name="name", description="description"
+        )
+        eid_name = uploader_name.create_experiment()
+        self.assertEqual(eid_name, "123")
+        eid_description = uploader_description.create_experiment()
+        self.assertEqual(eid_description, "123")
+        eid_both = uploader_both.create_experiment()
+        self.assertEqual(eid_both, "123")
```

Comment:
It would be nice to test that these actually send the proper requests.

Reply:
See "UpdateExperimentMetadataTest" below.   I think I added it after you were finished reviewing this section.

Reply:
Done.  Broke this into 3 tests.